composer.json

```diff
@@ -41,7 +41,7 @@
     }
   ],
   "require": {
-    "basho/riak": "^3.3",
+    "basho/riak": "dev-features/lrb/gset-crdt-RIAK-2907",
```

Comment:
@christophermancini How do you track the moving dependencies as these move towards release?